docs/integrations/data-ingestion/clickpipes/postgres/deduplication.md

```diff
@@ -0,0 +1,208 @@
+
+Below is an example of a CREATE Table statement executed by ClickPipes to create the table in ClickHouse.
+
+```sql
+CREATE TABLE users
+(
+    `id` Int32,
+    `reputation` String,
+    `creationdate` DateTime64(6),
+    `displayname` String,
+    `lastaccessdate` DateTime64(6),
+    `aboutme` String,
+    `views` Int32,
+    `upvotes` Int32,
+    `downvotes` Int32,
+    `websiteurl` String,
+    `location` String,
+    `accountid` Int32,
+    `_peerdb_synced_at` DateTime64(9) DEFAULT now64(),
+    `_peerdb_is_deleted` Int8,
+    `_peerdb_version` Int64
+)
+ENGINE = ReplacingMergeTree(_peerdb_version)
+PRIMARY KEY id
+ORDER BY id;
+```
+
+### Illustrative example {#illustrative-example}
+
+The illustration below walks through a basic example of synchronization of a table `users` between PostgreSQL and ClickHouse using ClickPipes.
+
+<Image img={clickpipes_initial_load} alt="Clickpipes initial load" size="lg" border/>
+
+**Step 1** shows the initial snapshot of the 2 rows in PostgreSQL and ClickPipes performing the initial load of those 2 rows to ClickHouse. If you observe, both rows are copied as-is to ClickHouse.
+
+**Step 2** shows three operations on the users table: inserting a new row, updating an existing row, and deleting another row.
+
+**Step 3** shows how ClickPipes replicates the INSERT, UPDATE, and DELETE operations to ClickHouse as versioned inserts. The UPDATE appears as a new version of the row with ID 2, while the DELETE appears as a new version of ID 1 with is _deleted marked as true. Because of this, ClickHouse has three additional rows compared to PostgreSQL.
+
+As a result, running a simple query like `SELECT count(*) FROM users;` may produce different results in ClickHouse and PostgreSQL. According to the [ClickHouse merge documentation](/merges#replacing-merges), outdated row versions are eventually discarded during the merge process. However, the timing of this merge is unpredictable, meaning queries in ClickHouse may return inconsistent results until it occurs.
+
+How can we ensure identical query results in both ClickHouse and PostgreSQL?
+
+###  Deduplicate using FINAL Keyword {#deduplicate-using-final-keyword}
+
+The recommended way to deduplicate data in ClickHouse queries is to use the [FINAL modifier.](/sql-reference/statements/select/from#final-modifier) This ensures only the deduplicated rows are returned.
+
+Let's look at how to apply it to three different queries.
+
+_Note in the following queries the WHERE clause to filter out deleted rows._
+
+- **Simple count query**: Count the number of posts.
+
+This is the simplest query you can run to check if the synchronization went fine. The two queries should return the same count.
+
+```sql
+-- PostgreSQL
+SELECT count(*) FROM posts;
+
+-- ClickHouse 
+SELECT count(*) FROM posts FINAL where _peerdb_is_deleted=0;
```

Comment:
```sql
SELECT count(*) 
FROM posts 
FINAL 
-- highlight-next-line
WHERE _peerdb_is_deleted=0;
```

This can be used to highlight within code blocks: https://docusaurus.io/docs/markdown-features/code-blocks#highlighting-with-comments